.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
     hooks:
     -   id: rst
         name: rst
-        entry: rst-lint --encoding utf-8
+        entry: rst-lint
```

Comment:
I removed this because it sees the ``--encoding`` was removed, on my local it was raising an error